### PR TITLE
Fix README

### PR DESCRIPTION
**Pull request by grische**
Readme fixed by @andrewachen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
 ---
 
-###Installation
+### Installation
 
-* Download ZIP or via Github client.
-* Install AMDAPPSDK/OpenCL library (required for OpenCL functions)
-* `sudo apt-get install libpci-dev`
-* Unzip or git clone
+* Download and unzip or git clone: `git clone https://github.com/sling00/amdmeminfo`
```
Comment (nerdralph) [inline README.md:10]: Wrong URL.

Comment (nerdralph) [reply README.md:10]: I think it is best to leave out the URL, leaving, "Download and unzip or git clone."

Comment (grische) [reply README.md:10]: Fixed in #2

**Comment by grische** [comment]
Closing in favour of #2

**Pull request closed without merging**